db/operations.go

```diff
@@ -148,9 +163,8 @@ var reflectTypeTime = reflect.TypeOf(time.Time{})
 // Format based on type, value returned unescaped
 func normalizeValueType(value string, valueType reflect.Type) (string, error) {
 	switch valueType.Kind() {
-	case reflect.String:
+	case reflect.String, reflect.Slice:
```

Comment:
That's the last nitpick. I would like that we get `reflect.Slice` is own `case` and that we add a comment that explains that this exists when the schema defines a composite key (at least that's what I assume it is).

Reply:
Actually, this is a missing code for another PR.

When trying to use BYTES column in postgres, substreams uses Vec<u8> and when we convert to Go, it becomes a Slice. Postgres needs an escaped bytes the same as strings.

Sorry for this code, I'm opening another PR for that.